query/src/lib.rs

```diff
@@ -75,13 +76,20 @@ impl FromValue<Variable> for Variable {
 }
 
 impl Variable {
-    pub fn from_symbol(sym: &PlainSymbol) -> Option<Variable> {
```

Comment:
But here, you could take anything that's `Into<Rc<PlainSymbol>>`, since `Rc<T>: From<T>`.  I guess you want `try_into`?